client/lib/notifications/test/note-block-parser.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
```

Comment:
We're trying to migrate away from `chai` and use only Jest native expects in new code. In this test, you can use:
- `expect( x ).toEqual( y )` to perform a deep comparison
- `expect( x ).not.toEqual( y )` to reverse the condition
- `expect( x ).toHaveProperty( 'url' )` to check for existence of an object property